src/amo/components/LandingPage.js

```diff
@@ -108,9 +115,22 @@ export class LandingPageBase extends React.Component {
   }
 }
 
-export function mapStateToProps(state, ownProps) {
+export function mapStateToProps(
+  state, ownProps, _apiAddonType = apiAddonType, _log = log
+) {
+  let addonType;
+  try {
+    addonType = _apiAddonType(ownProps.params.visibleAddonType);
+  } catch (err) {
+    if (err.message.match('not found in API_ADDON_TYPES_MAPPING')) {
```

Comment:
Ah I guess since we compile to ES5 it should be fine.

Personally I'm for maintaining broad-as-possible browser support, as in the future Edge in particular should support Web Extensions. I'm not sure if this is an explicit product goal, but the idea of us being able to serve Edge and other browsers Web Extensions seems awesome to me. And I like us being good web advocates and making stuff work everywhere 😄 

Still, should work, I'll give it a go.